news/1082.feature

```diff
@@ -0,0 +1 @@
+This change enables deep merging for ListConfigs. Usage: `OmegaConf.merge(cfg1, cfg2, extend_lists=True)`
```

Comment:
```suggestion
This change enables deep merging (and optionally retaining duplicate values) for ListConfigs. Usage: `OmegaConf.merge(cfg1, cfg2, extend_lists=True)`
```

Reply:
I decided to choose a shorter sentence and took omry's suggestion. I think for the news summary, it makes sense to announce the feature, but keep it short. Users, that have interest in this feature, read the docs anyway